src/libkomwm.py

```diff
@@ -104,6 +105,9 @@ def mwm_encode_color(colors, st, prefix='', default='black'):
     if prefix:
         prefix += "-"
     opacity = hex(255 - int(255 * float(st.get(prefix + "opacity", 1))))
+    # TODO: Refactoring idea: here color is converted from float to hex. While MapCSS class
+    #       reads colors from *.mapcss files and converts to float. How about changing MapCSS
+    #       to keep hex values and avoid Hex->Float->Hex operations?
```

Comment:
we specify colors in *.mapcss via hex notation, aren't we?

though other notations are supported also, e.g. `color: red;`

Reply:
@strump could you comment here please?

It seems to me that `color = whatever_to_hex()` is used here in case a color isn't defined in hex format already, its not necessary float.

Maybe its redundant to require the mapcss.webcolors lib indeed. We use hex notation almost everywhere in styles. Sometimes could be useful to e.g. use simple names like red, green, etc. But not necessary.